trustly-android-sdk/src/main/java/net/trustly/android/sdk/views/TrustlyView.java

```diff
@@ -329,6 +329,11 @@ public Trustly establish(Map<String, String> establishData) {
             data.put("cancelUrl", cancelURL);
             data.put("grp", Integer.toString(grp));
 
+            String allowedPaymentProviderTypes = establishData.get("allowedPaymentProviderTypes");
+            if (allowedPaymentProviderTypes != null) {
+                data.put("allowedPaymentProviderTypes", allowedPaymentProviderTypes);
```

Comment:
In the task description we are mentioned only the **_allowedPaymentProviderTypes_**  property, but I think we should consider all the new fields in the settings screen, to check if they already is in the SDK, like _**onlinePPSubtype, manualPPSubtype and kycType**_, fields.